LectureNotes/Lectures/Fall2018/Lecture4/PPP_notes.tex

```diff
@@ -0,0 +1,186 @@
+	\end{align}
+	\item \emph{Thinning:} Independently discard each point of $N$ with probability $1-p(x)$ for a point at $x\in X$. The result is a $\operatorname{PPP}(\nu)$, where
+	\begin{align}\label{eq:thin}
+		\nu(A) = \int_A\! p(x) \, d\mu(x).
+	\end{align}
+	In other words, if $N = \sum_i \delta_{x_i}$ and $a_i = 1$ with probability $p(x_i)$ and $a_i = 0$ otherwise, then
+	\begin{align}
+		\widetilde{N} = \sum_i a_i \delta_{x_i} \sim \operatorname{PPP}(\nu).
+	\end{align}
+	\item \emph{Additivity:} If $N_1 \sim \operatorname{PPP}(\mu_1)$ and $N_2 \sim \operatorname{PPP}(\mu_2)$ are independent on $X$, then $N_1 + N_2 \sim \operatorname{PPP}(\mu_1 + \mu_2)$. That is to say, if $\mathbb{P}\left\{ X = n \right\} = \frac{\lambda^n}{n!}e^{-\lambda}$ and $\mathbb{P}\left\{ Y = n \right\} = \frac{\nu^n}{n!}e^{-\nu}$ are independent, then
+	\begin{align}
+		\mathbb{P}\left\{ X + Y = n \right\} = \frac{(\lambda + \nu)^n}{n!}e^{-(\lambda + \nu)}.
+	\end{align}
+	\item \emph{Labeling:} For each point in a PPP, associate an independent label from a space $Y$ according to some distribution $\nu$. Let $N = \sum_i \delta_{x_i}$ for $\{x_i\}\subset X$ and let $G_1,G_2,\ldots \in Y$ be iid with density $\nu$. Then 
+	\begin{align}\label{eq:label}
+		\overline{N} := \sum_i \delta_{(x_i, G_i)} \sim \operatorname{PPP}(\mu\times\nu)
+	\end{align}
+	on $X\times Y$.
+\end{itemize}
+\section{Examples}
+Henceforth, let $\lambda$ denote Lebesgue measure.
+\begin{example}
+	Let $N\sim \operatorname{PPP}(\lambda)$ on $\mathbb{R}_{\geq 0}$, where $\lambda$ is Lebesgue measure.
+	\begin{enumerate}[(a)]
+		\item How far until the first light?
+		\item Suppose each light is independently either red or green with probability $\frac{1}{2}$. How far until the first red light?
+	\end{enumerate}
+	\begin{soln}
+		Let $N = \sum_i \delta_{x_i}$ and put $T = \min\{x_i\}$. Using \eqref{eq:ppp} we compute
+		\begin{align}
+			\mathbb{P}\left\{ T > t \right\} = \mathbb{P}\left\{ N([0,t]) = 0 \right\} = e^{-t}.
+		\end{align}
+		This solves part (a). For the colorblind readers, this also solves part (b). 
+		
+		Now let $\{\tilde{x}_i\}\subset \{x_i\}$ be the (random) set of red lights and define $\widetilde{N} = \sum_i \delta_{\tilde{x}_i}$, the point process for the red lights from $N$. By the thinning property \eqref{eq:thin}, $\widetilde{N} \sim \operatorname{PPP}\left(\frac{1}{2}\lambda\right)$. Similarly define $\widetilde{T} = \min\{\tilde{x}_i\}$ and observe
+		\begin{align}
+			\mathbb{P}\left\{ \widetilde{T} > t\right\} = \mathbb{P}\left\{ \widetilde{N}([0,t]) = 0 \right\} = e^{-t/2},
+		\end{align}
+		thus (b) is solved.
+	\end{soln}
+\end{example}
+\begin{example}\label{ex:ant}
+	Rain falls for 10 minutes on a large patio at a rate of $\nu = 12$ drops per minute per square meter. Each drop splatters to a radius $R \sim \operatorname{Exp}(100)$, so that $\mathbb{E}[R] = 1\text{ cm}$, with a thickness of $0.001$ meters. Assume the set of locations of the raindrops is a PPP.

```

Comment:
maybe say that "each drop splatters to a random radius that has an Exponential distribution, with mean 1cm, independently of the other drops"  (the "100" is hard to interpret)

Reply:
How about change this to 12 drops per square cm? Then we'll have $\nu = 1.2 \times 10^5$ per $m^2$, making the numbers below not tiny.

Reply:
> How about change this to 12 drops per square cm? Then we'll have $\nu = 1.2 \times 10^5$ per $m^2$, making the numbers below not tiny.

Not sure how much time we should spend honing this parameter. 12 drops per minute per square cm will give probability the ant is dry on the order of 1e-33. This makes sense, given the splatter radii. I suggest nu = 4700 drops per minute per square meter for around 5% chance the ant is dry, or nu = 82 drops per minute per square meter for about 95% chance the ant is dry.

Reply:
Either way. 5000 per m^2 = 0.5 per cm^2 would be close to your first number; how about that?